### DISCUSSION TITLE
Disabling environment deploy notifications

### DISCUSSION THREAD

**Discussion by rainest** [community/community#38618] [Actions]
We use environments exclusively to fence secrets to the specific jobs that need them. We aren't using them to model an actual deployment environment (testing, production, etc.).

When a CI workflow attached to an environment runs on a PR, it adds a timeline event to the PR for the individual jobs in that workflow, for each individual run. One of our workflows uses quite a large number of jobs, as it's a matrix of different tests against different versions. This results in _a lot_ of timeline events, especially if the workflow runs more than once:

![environment](https://user-images.githubusercontent.com/571832/200706248-7811a735-009f-48ae-96a3-dd5228bb7aaf.png)
(from https://github.com/Kong/kubernetes-ingress-controller/pull/3035).

These events aren't particularly useful for us, as we're usually only checking the latest runs in the checks dialog, or are checking past runs via the Actions tab. We don't have much use for an event for every individual run. The events are instead actively harmful:
- We have to scroll past a large number of events to get to the checks dialog/new comment/merge/etc. that come after them.
- More useful timeline events, such as reviewer comments, get lost in a large pile of hidden items.
- Sufficiently many events (as in the linked PR) are enough to start slowing down the Firefox UI.

Is there a way to configure an environment such that it does not add these events to PRs? If not, can one be added?

**Comment by ossanna16** [comment]
Hi there @rainest and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
